src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { type IconProps } from "./IconBase";
+export * from "./icons/index";
```

Comment:
I prefer we don't use any type of barrel export and instead we generate here every single export